test/cmd/secrets.sh

```diff
@@ -85,6 +85,20 @@ os::cmd::expect_success 'oc secrets add deployer basicauth sshauth --for=pull'
 os::cmd::expect_success 'oc secrets add deployer basicauth sshauth --for=pull,mount'
 
 # attach secrets to service account
+# test that those secrets can be unlinked
+# after they have been deleted.
+os::cmd::expect_success 'oc create secret generic deleted-secret'
+os::cmd::expect_success 'oc secrets link deployer deleted-secret'
+# confirm our soon-to-be-deleted secret has been linked
+os::cmd::expect_success_and_text "oc get serviceaccount deployer -o jsonpath='{.secrets[?(@.name==\"deleted-secret\")]}'" 'deleted\-secret'
+os::cmd::expect_success 'oc get serviceaccounts/deployer -o yaml |grep -q deleted-secret'
```

Comment:
So when you unlink you should see no text here

Reply:
In the JSONpath that is -- there should be no yaml grep in the final tests

Reply:
Yeah, this is testing that the secret has linked. A few lines below from this, I test unlinking by calling `os::cmd::expect_success_and_not_text`

Reply:
Also, forgot to remove the old `grep` check. Will remove that